plugins/available/pack.plugin.bash

```diff
@@ -568,6 +569,7 @@ _pack_root_command() {
 __start_pack() {
 	local cur prev words cword
 	declare -A flaghash 2> /dev/null || :
+	export flaghash
```

Comment:
What is this "export" for? Bash variables are dynamic scoping so are visible from callees regardless of the export attribute. The export attribute is used to pass values to external commands.